src/masternodes/mn_rpc.cpp

```diff
@@ -1558,32 +2005,675 @@ UniValue accounttoutxos(const JSONRPCRequest& request) {
+            UniValue paginationObj = request.params[0].get_obj();
+            if (!paginationObj["limit"].isNull()) {
+                limit = (size_t) paginationObj["limit"].get_int64();
+            }
+            if (!paginationObj["start"].isNull()) {
+                including_start = false;
+                start.v = (uint32_t) paginationObj["start"].get_int();
+            }
+            if (!paginationObj["including_start"].isNull()) {
+                including_start = paginationObj["including_start"].getBool();
+            }
+            if (!including_start) {
+                ++start.v;
+            }
+        }
+        if (limit == 0) {
+            limit = std::numeric_limits<decltype(limit)>::max();
+        }
+    }
+
+    LOCK(cs_main);
+
+    PoolShareKey startKey{ start, CScript{} };
+//    startKey.poolID = start;
+//    startKey.owner = CScript(0);
+
+    UniValue ret(UniValue::VOBJ);
+    pcustomcsview->ForEachPoolShare([&](DCT_ID const & poolId, CScript const & provider) {
+        const CTokenAmount tokenAmount = pcustomcsview->GetBalance(provider, poolId);
+        if(tokenAmount.nValue) {
+            const auto poolPair = pcustomcsview->GetPoolPair(poolId);
+            if(poolPair) {
+                ret.pushKVs(poolShareToJSON(poolId, provider, tokenAmount.nValue, *poolPair, verbose));
+            }
+        }
+        limit--;
+        return limit != 0;
+    }, startKey);
+
+    return ret;
+}
+
+UniValue listcommunitybalances(const JSONRPCRequest& request) {
+    RPCHelpMan{"listcommunitybalances",
+               "\nReturns information about all community balances.\n",
+               {
+               },
+               RPCResult{
+                       "{balance_type:value,...}     (array) Json object with accounts information\n"
+               },
+               RPCExamples{
+                       HelpExampleCli("listcommunitybalances", "")
+                       + HelpExampleRpc("listcommunitybalances", "")
+               },
+    }.Check(request);
+
+    /// @todo is it important to restrict with `Params().GetConsensus().AMKHeight` ??
+
+    UniValue ret(UniValue::VOBJ);
+
+    LOCK(cs_main);
+    for (auto kv : Params().GetConsensus().nonUtxoBlockSubsidies) {
+        ret.pushKV(GetCommunityAccountName(kv.first), ValueFromAmount(pcustomcsview->GetCommunityBalance(kv.first)));
+    }
+
+    return ret;
+}
+
+UniValue setgov(const JSONRPCRequest& request) {
+    CWallet* const pwallet = GetWallet(request);
+
+    RPCHelpMan{"setgov",
+               "\nSet special parameters.\n",
+               {
+                    //{"govKey", RPCArg::Type::STR, RPCArg::Optional::NO,"First parameter - governance key."},
+                    {"parameters", RPCArg::Type::OBJ, RPCArg::Optional::NO, "",
+                        {
+                             {"name", RPCArg::Type::STR, RPCArg::Optional::NO,
+                              "Second parameter - governance data."},
```

Comment:
If the first parameter is commented, then this is the first parameter already

Reply:
Thanks. Description was not correct in fact. See newer version